app/helpers/application_helper.rb

```diff
@@ -1,2 +1,47 @@
 module ApplicationHelper
+
+  def recreation_list
+    [
+      [' '],
+      ['Ski'],
+      ['Board'],
+      ['Both'],
+      ['Other']
+    ]
+  end
```

Comment:
Is it possible to have a text box go along with 'Other'? If it's an easy-ish update, I think that would be nice for inclusivity

Reply:
Ive got other listed? or are we talking about a different kind of other?

Reply:
Yeah the way I was looking at it is that there's an 'Other' option in the dropdown, but if you are putting that other, it might be nice to be able to specify what your other is. Cause something like a monoski takes up a lot less room in a car than something like one of those ski bikes.